### PR TITLE
Set up

### PR DESCRIPTION
**Pull request by andgarzonmal**
- [ ] Create bookcontainer component 
- [ ] Create book component
- [ ] Create form componenr 
- [ ] Create nav copmonet
- [ ] Set up routers

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          path="/Categories"
+          element={(
+            <div className="App">
+              <Categories />
+            </div>
+        )}
+        />
+      </Routes>
+
+    </BrowserRouter>
```
Comment (mateo951) [inline src/App.js:32]: `react-router-dom` is present in `package.json` and used to make the routes 🔥

File: src/components/BookList.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  <li>
+    <div>
+      <h1>{title}</h1>
+      <h2>{author}</h2>
+      <ul>
+        <li>
+          <button type="button">Comments</button>
+        </li>
+        <li>
+          <button type="button">Remove</button>
```
Comment (mateo951) [inline src/components/BookList.js:14]: Has a button for removing books ✔

File: src/components/Form.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export default function Form() {
+  return (
+    <div>
+      <h1>ADD NEW BOOK</h1>
+      <input type="text" placeholder="book title" />
```
Comment (mateo951) [inline src/components/Form.js:7]: Please be sure that your form also has an input for author 🧐

**Review by mateo951 — CHANGES_REQUESTED** [review]
Hi @andgarzonmal, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: src/components/Form.js (hunk 3 above)
Comment (andgarzonmal) [reply src/components/Form.js:7]: You are correct, thanks.

**Review by rustam-tolipov — APPROVED** [review]
## Hello @andgarzonmal 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**